src/main/startup/system_stm32h7xx.c

```diff
@@ -597,26 +626,48 @@ void SystemClock_Config(void)
     HAL_RCCEx_PeriphCLKConfig(&RCC_PeriphClkInit);
 
 #ifdef USE_SDCARD_SDIO
+    __HAL_RCC_SDMMC1_CLK_ENABLE(); // FIXME enable SDMMC1 or SDMMC2 depending on target.
```

Comment:
Are we going to fix this now?

Reply:
no, it's beyond the scope of this PR.  The authors of the code that added SDMMC2 support should have addressed it.